ipn/prefs.go

```diff
@@ -44,6 +45,24 @@ type Prefs struct {
 	// packets stop flowing. What's up with that?
 	AllowSingleHosts bool
 
+	// ExitNodeID and ExitNodeIP specify the node that should be used
+	// as an exit node for internet traffic. At most one of these
```

Comment:
"at most one" ... where is that enforced? I might've missed it.

Reply:
Not enforced directly, but if ExitNodeIP is non-zero, it takes precedence and will eventually overwrite ExitNodeID.